File: Scala/Top3.scala
object Top3 extends App{
    val list1: List[Int] = List(25, 36, 4, 55, 71, 18, 0, 71, 89, 65);
    top3(list);

    /**
     * 配列の中でTOP3の値を取得する
     */
    def top3(list: List[Int]) = {
        var editedList: List[Int] = list;
        var maxValue = 0;
        for (i <- 0 until 3) {
            maxValue = editedList.max;
            editedList = editedList.diff(List(maxValue));
            println(maxValue);
        };
    };
}

File: JavaScript/JavaScript/Top3.js
/**
 * 配列の中でTOP3の値を取得する
 */
function top3() {
  const list = [25, 36, 4, 55, 71, 18, 0, 71, 89, 65];

  for (let i = 0; i < 3; i++) {
    const maxValue = Math.max(...list);
    const deletedIndexNumber = list.indexOf(maxValue);
    list.splice(deletedIndexNumber, 1);
    console.log(maxValue);
  }
}